src/Microsoft.AspNet.Razor.Runtime/TagHelperContent.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Abstract class which is used to buffer content returned by <see cref="TagHelpers.ITagHelper"/>.
```

Comment:
comments in this class need some updates:
- end this sentence with an "s"
- either "Gets an value indicating whether the content was modified." (and update the other `bool` properties) or `Returns <c>true</c> if the <see cref="TagHelperContent"/> has been modified; <c>false</c> otherwise.` (the first is more consistent with properties in MVC; the second requires less change in this file.)
- nit: leave out most or all `of <see cref="TagHelperContent"/>` bits.
- reword sentences that use "to which" or "must".  in other words, be direct e.g. `A <see cref="string"/> that replaces the content.`